lua/ide/components/explorer/filenode.lua

```diff
@@ -152,10 +152,28 @@ FileNode.new = function(path, kind, perms, depth, opts)
                     return
                 end
             end
-            if vim.fn.writefile({}, path) == -1 then
-                error("failed to write file " .. path)
-                return
+
+            -- if path ends with '/' create a directory instead
+            if vim.endswith(path, '/') then
```

Comment:
instead of reimplementing `self.mkdir()` can you use that method on the filenode node instead?

Reply:
sure, done.